### PR TITLE
update release documentation and patches

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- remove patch supporting for epel6
- update patch for epel7 to remove ocp4 product, it can't be built in epel7
- update documentation with above changes
- describe stabilization phase in documentation

#### Rationale:

- _Rationale here. Replace this text. Don't use the italics format!_

- Fixes # _Issue number here (e.g. #26) or remove this line if no issue exists._

### CODE REVIEW

File: release_tools/README.md [hunk 1]
```diff
@@ -56,6 +56,36 @@ developers](https://github.com/orgs/ComplianceAsCode/teams/trusted-developers).
 >     owner: "--owner <owner of repo>" # your github user name
 >     repo: "--repo <name of repo>"    # your clone's repo name
 
+Stabilization phase
+----------------------------------------------------------
+It is a good practice to have a stabilization phase before the upstream release
+is performed. During this period, the new stabilization branch is created. Only
+bugfixes which are discovered during this stabilization period are cherry-picked
+from the master branch into the stabilization branch. This reduces risk of
```
Comment (carlosmmatos) [inline release_tools/README.md:64]: This reduces the risk of

File: release_tools/README.md [hunk 2]
```diff
@@ -56,6 +56,36 @@ developers](https://github.com/orgs/ComplianceAsCode/teams/trusted-developers).
...
+bringing a sudden breaking change into the release.
+
+We usually have stabilization phase two weeks long. To start the stabilization:
+
+- create a new branch called stabilization-vx.y.z, where x.y.z is the version of
```
Comment (carlosmmatos) [inline release_tools/README.md:69]: Capitalize first character

File: release_tools/README.md [hunk 3]
```diff
@@ -56,6 +56,36 @@ developers](https://github.com/orgs/ComplianceAsCode/teams/trusted-developers).
...
+
+During the stabilization:
+
+- if a bug is fixed, cherry-pick the PR from the master branch into the
+  stabilization branch
+
+- change the milestone for the cherry-picked PR to the milestone of upcoming
+  release
+
+- label the PR with "backported_into_stabilization"
```
Comment (carlosmmatos) [inline release_tools/README.md:86]: Make sure each bullet `-` starts is capitalized for consistency.

**Review by carlosmmatos — CHANGES_REQUESTED** [review]

File: release_tools/copr_epel7.patch [hunk 4]
```diff
@@ -13,11 +13,12 @@
  Requires:	xml-common, openscap-scanner >= 1.2.5
  Obsoletes:	openscap-content < 0:0.9.13
  Provides:	openscap-content
-@@ -41,13 +41,16 @@
+@@ -41,13 +41,17 @@
  
  %prep
  %setup -q
 +mkdir build
++sed -i 's/option(SSG_PRODUCT_OCP4 "If enabled, the OCP4 SCAP content will be built" \${SSG_PRODUCT_DEFAULT})/option(SSG_PRODUCT_OCP4 "If enabled, the OCP4 SCAP content will be built" false)/' CMakeLists.txt
```
Comment (yuumasato) [inline release_tools/copr_epel7.patch:21]: It should be safer to run the  `cmake` command with `-DSSG_PRODUCT_OCP4:BOOLEAN=OFF`
` cmake -DSSG_PRODUCT_OCP4:BOOLEAN=OFF ..`

**Review by yuumasato — CHANGES_REQUESTED** [review]

**Comment by vojtapolasek** [comment]
Thank you for feedback, fixed.

**Review by yuumasato — APPROVED** [review]

**Comment by yuumasato** [comment]
Thank you for documenting this.

**Pull request merged**